src/main/java/at/hannibal2/skyhanni/features/chat/ChatFilter.kt

```diff
@@ -464,8 +460,8 @@ object ChatFilter {
         "solo_stats" to soloStatsPatterns,
         "fairy" to fairyPatterns,
         "achievement_get" to achievementGetPatterns,
-        "parkour" to parkourPatterns,
-        "teleport_pads" to teleportPadPatterns,
+        "hideParkour" to parkourPatterns,
+        "hideTeleportPads" to teleportPadPatterns,
```

Comment:
everythign elese here has underscores so this camelCase seems out of place

Reply:
just copied what hannibal changed, thought it was best to not go against that

https://github.com/hannibal002/SkyHanni/pull/2348/commits/473f962286372e0e09fea46a355e12c00fcd2589